### ISSUE TITLE
[Sage Discussion]: Host-guest/protein ligand binding data

### ISSUE THREAD

**Issue by ocmadin** [openforcefield/nistdataselection#18]
Host-guest/protein ligand binding data: Do we use host-guest/protein-ligand data in our fitting? Or just for benchmarking? If so, which systems should we look at?

**Comment by davidlmobley** [comment]
I don't think we're ready to fit on it yet until we have a set of improved LJ parameters that uses solution-phase properties only. I'm less confident as to what will happen when we begin including host-guest data, so I'd like to bring that in gradually in a controlled manner.

**Comment by ocmadin** [comment]
Yeah, if we're targeting an early May release using host-guest/protein-ligand in fitting isn't really feasible.  We should consider it for benchmarking, though.
